priam/src/main/java/com/netflix/priam/notification/BackupNotificationMgr.java

```diff
@@ -89,7 +94,7 @@ public void notify(BackupVerificationResult backupVerificationResult) {
     }
 
     private Map<String, MessageAttributeValue> getMessageAttributes(
-            AbstractBackupPath.BackupFileType backupFileType) {
+            AbstractBackupPath.BackupFileType backupFileType, JSONObject additionalAttributes) {
         // SNS Attributes for filtering messages. Cluster name and backup file type.
         Map<String, MessageAttributeValue> messageAttributes = new HashMap<>();
         messageAttributes.putIfAbsent(
```

Comment:
As referenced above these 10 lines resolve to:

messageAttributes.put("s3clustername", toStringAttribute(message.get("s3clustername")));
messageAttributes.put("backuptype", toStringAttribute(message.get("backuptype")));